src/core/migrations/0046_delete_review_request_sent.py

```diff
@@ -0,0 +1,75 @@
+# -*- coding: utf-8 -*-
+# Generated by Django 1.11.29 on 2020-12-21 13:59
+from __future__ import unicode_literals
+
+from django.db import migrations
+
+
+def delete_settings(apps, schema_editor):
+    Setting = apps.get_model('core', 'Setting')
+    Setting.objects.filter(
+        group__name='email',
+        name='review_request_sent',
+    ).delete()
+
+
+def update_setting_values(apps, schema_editor):
+    SettingValueTranslation = apps.get_model('core', 'SettingValueTranslation')
+
+    prod_complete_settings = SettingValueTranslation.objects.filter(
+        master__setting__name='production_complete',
+        master__setting__group__name='email',
+    )
+
+    for setting in prod_complete_settings:
+        setting.value = setting.value.replace(
+            '<p>This article is now in the Proofing workflow: {{ proofing_list_url }}.</p>',
+            '',
+        )
+        setting.value = setting.value.replace(
+            'Dear {{ assignment.editor.full_name }},</p><p>This is an automatic',
+            'This is a',
+        )
+        setting.save()
+
+    typesetter_notification_settings = SettingValueTranslation.objects.filter(
+        master__setting__name='typesetter_notification',
+        master__setting__group__name='email',
+    )
+
+    for setting in typesetter_notification_settings:
+        setting.value = setting.value.replace(
+            'This is an automatic',
+            'This is a',
+        )
+        setting.save()
+
+    notify_editor_proofing_complete_settings = SettingValueTranslation.objects.filter(
+        master__setting__name='notify_editor_proofing_complete',
+        master__setting__group__name='email',
+    )
+
+    for setting in notify_editor_proofing_complete_settings:
+        setting.value = setting.value.replace(
+            ': {{ publish_url }}',
+            '',
```

Comment:
This change is not reflected in the defaults

Reply:
Default is:

```
<p>Dear {{ proofing_assignment.editor.full_name }},</p><p>Proofing for "{{ article.title }}" is now complete.</p><p>Regards,</p><p>{{ request.user.signature|safe }}</p>
```
{{ publish_url }} has been removed as far as I can tell.

Reply:
Why do we need a migration to delete it then? Is it present on some installs?

Reply:
Yes - it certainly was in mine.